packages/components/src/Card/Card.stories.tsx

```diff
@@ -19,55 +15,6 @@ const meta = {
       description: 'The variant of the card',
       options: ['default', 'master', 'admin'],
     },
-    size: {
-      control: {
-        type: 'radio',
-      },
-      description: 'The size of the card',
-      options: ['default', 'stretch', 'small'],
-    },
-    p: {
-      control: {
-        type: 'text',
-      },
-      description: 'The padding of the card',
-    },
-    px: {
-      control: {
-        type: 'text',
-      },
-      description: 'The x padding of the card',
-    },
-    py: {
-      control: {
-        type: 'text',
-      },
-      description: 'The y padding of the card',
-    },
-    pr: {
-      control: {
-        type: 'text',
-      },
-      description: 'The right padding of the card',
-    },
-    pt: {
-      control: {
-        type: 'text',
-      },
-      description: 'The top padding of the card',
-    },
-    pb: {
-      control: {
-        type: 'text',
-      },
-      description: 'The bottom padding of the card',
-    },
-    pl: {
```

Comment:
somehow the controls in the storybook are not working, or at least I can't change anything

Reply:
same for `stretch`

Reply:
stretch works when I test it 🤔